.github/workflows/release.yml

```diff
@@ -115,7 +115,9 @@ jobs:
       - name: Publish reference docs
         run: |
           gem install jazzy
-          brew install sourcekitten
+          brew tap-new $USER/local-sourcekitten
```

Comment:
We _assume_ `$USER/local-sourcekitten` will work as expected but were not able to find anything online confirming one way or the other. This is the last step so if for some reason this blows up we should look at maybe moving the docs into its own workflow that can be run independent of the release script if needed. Thoughts?